linux/kernel/updating.md

```diff
@@ -1,12 +1,10 @@
 # Updating the Kernel
 
-If you are using the Raspberry Pi stock kernel, your system can be kept up to date by the `rpi-update` utility:
+Most distributions for the Raspberry Pi provide their own packages for an updated kernel. It is highly recommended to update your kernel as part of your regular system updates instead of manually installing one.
 
-```
-$ sudo rpi-update
-```
+If you are using NOOBS to run Raspbian, the Raspberry Pi Foundation kernel is part of the installed `raspberrypi-bootloader` package. This package also contains the nessecary bootloader files for the Broadcom chip. If you are running Debian/ARM (Raspberry Pi 2) or a custom Rasbian install, the kernel might be installed via the `linux-image` package with the Broadcom bootloader files in the `raspberrypi-bootloader-nokernel` package.
```

Comment:
There's nothing NOOBS-specific about this - NOOBS installs the same version of Raspbian as can be found on https://www.raspberrypi.org/downloads/
And you've misspelled 'necessary'.

I'm not sure that mentioning Debian/ARM or 'custom Raspbian' is appropriate, as (AFAIK) the documentation on raspberrypi.org is explicitly only targeted at 'stock' Raspbian, in order to not confuse newbies.